### ISSUE TITLE
Create Databse tests

### ISSUE THREAD

**Issue by shailpatels** [Submitty/Submitty#7358]
**What problem are you trying to solve with Submitty**
It would be nice in certain scenarios to write tests that can query against a database. For example, running a database query in PHP and then checking if the database is in the correct state or has the expected data in it. 

**Describe the way you'd like to solve this problem**
These could be written in python and use sqlite as a driver and would work with the other selenium e2e tests, it would be nice however to get PHP e2e tests then certain parts of the PHP codebase code be directly run

**Describe any potential alternatives you'd tried to solve the problem**
Tests would need to clean up after themselves or run against a database that can run in memory or be removed and recreated

**Comment by wusatosi** [comment]
I don't think you need a driver written in python to test the database operations, I think writing unit tests for [DatabaseQueries.php](https://github.com/Submitty/Submitty/blob/d384373466afaaea65f2adcd0cf23963581e471a/site/app/libraries/database/DatabaseQueries.php) is sufficient enough/ a good way to start. I am happy to help once I'm free.

**Comment by MasterOdin** [comment]
What exactly are we trying to verify here? I would caution against spending too much time on this task given that the database queries file should eventually go away as we move more and more to doctrine.

**Comment by MasterOdin** [comment]
> use sqlite as a driver 

I'm confused by this, but I would say many of our queries use postgresql specific features and would not run on sqlite.

**Comment by shailpatels** [comment]
I think the gist of this would be some way to verify the state of the database is in a specific way after running some PHP code against an actual db. This would for verifying an action like for late day's making sure the triggers work as expected for adding or removing entries but could be used to verify other actions like sending out emails, or creating users.

If there is a better way to test this or if its not worth setting up automated tests for this issue let me know, I haven't had time to put real work into building something for this yet
